src/scenes/level-selection-scene.ts

```diff
@@ -10,21 +10,19 @@ import { AudioPlayer } from "@components";
 import { getData, GameScore } from "@data";
 import { SelectedLevel } from "../Firebase/firebase-event-interface";
 import { FirebaseIntegration } from "../Firebase/firebase-integration";
-import {
-  createBackground,
-  levelSelectBgDrawing,
-} from "@compositions/background";
+import { createBackground, levelSelectBgDrawing } from '@compositions/background';
```

Comment:
lets use double quotes just to be consistent for imports

Reply:
Changes applied, thank you.